indium-webkit.el

```diff
@@ -224,7 +228,18 @@ Allowed states: `\"none\"', `\"uncaught\"', `\"all\"'."
                                       nil
                                       t)))
   (indium-webkit--send-request `((method . "Debugger.setPauseOnExceptions")
-                               (params . ((state . ,state))))))
+                                 (params . ((state . ,state))))))
+
+(defun indium-webkit--set-cache-disabled (disabled)
+  "Toggle ignoring cache for each request.  If DISABLED, cache will not be used."
+  (indium-webkit--send-request `((method . "Network.setCacheDisabled")
+                                 (params . ((cacheDisabled . ,(if disabled t :json-false)))))))
+
+(defun indium-webkit-set-cache-disabled (arg)
+  "Disable network cache for each request.  If prefix ARG is used enable it."
+  (interactive "p")
+  (setq indium-webkit-cache-disabled (= arg 1))
+  (indium-webkit--set-cache-disabled indium-webkit-cache-disabled))
```

Comment:
Using a prefix argument feels a bit weird IMO.

What about `indium-webkit-enable-cache` and `indium-webkit-disable-cache` instead?